packages/@aws-cdk/aws-glue-alpha/test/job.test.ts

```diff
@@ -1176,5 +1203,19 @@ describe('Job', () => {
         })).toThrow('Both workerType and workerCount must be set');
       });
     });
+
+    describe('validation for jobRunQueuingEnabled and maxRetries', () => {
+      test('jobRunQueuingEnabled with maxRetries > 0 should throw', () => {
```

Comment:
If there is only one test within a describe block, I think a single test block is sufficient.